src/error.rs

```diff
@@ -145,6 +162,15 @@ If none of the above steps work for you, please contact the following people on
                 ),
                 _ => None,
             },
+            CliError::DevConfigError(error) => match error {
+                DevConfigError::ConfigSecretNotFound => Some(
+                    "Run \"wukong config dev pull\" to pull the latest dev config.".to_string()
+                ),
+                DevConfigError::InvalidSecretPath { path } => Some(format!(
+                "Please check the secret path in the config file: config/{path}"
```

Comment:
this one is misleading. Since the secret path is declared by the annotation in the `dev.exs` file, we should not tell users to check the path in the secret file. 

```
comparing local config vs remote config...
Error - Invalid secret path in the config file
Suggestion -
        Please check the secret path in the config file: config/dev.secrets1.exs
```

The right way is telling user to check the secret path in the file that contains the annotation to the secret path.

Reply:
<img width="695" alt="Screenshot 2023-04-06 at 4 33 18 PM" src="https://user-images.githubusercontent.com/25294487/230321403-915af6eb-d8b7-46fb-b342-7e96da7ae83b.png">

updated with dev config file instead of secret file